src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+//new class Bulldozer
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped working");
```

Comment:
Similarly, the message 'Bulldozer stopped working' should be more informative. Consider rephrasing to 'Bulldozer has stopped working.' to clearly indicate the action and the type of Machine.